.github/workflows/automated_release.rc_version.yml

```diff
@@ -0,0 +1,44 @@
+name: "Automated Release: Release next RC from release branch"
+
+on:
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
I think we need a stricter naming convention for these branches - otherwise someone will create a branch 65.11.22 and it will get released

Reply:
Yes, I agree. I thought we could use `x.y.z-<branch cut sha>` as naming convention but it looked ugly. So blocking such creations should be fine. I will try to configure that as well as some merging restrictions in release branch